virttest/storage.py

```diff
@@ -783,6 +783,9 @@ def __init__(self, params, root_dir, tag):
         self.root_dir = root_dir
         self.base_tag = None
         self.snapshot_tag = None
+        self.bitmap_name = params.get("bitmap_name")
+        self.bitmap_granularity = params.get("bitmap_granularity", "65536")
+        self.bitmap_name_source = params.get("bitmap_name_source")
```

Comment:
an image can have multiple bitmaps, we cannot represent all with a single bitmap_name here;
granularity and source bitmap name are not members of a qemu image object, IMO, which are bitmap attributes
So remove all

Totally, we introduce two new bitmap level params:
bitmap_image: image tag name where the bitmap locates, e.g. image1
bitmap_granularity: granularity of the bitmap

We can set different granularities for different bitmaps
bitmap_granularity_bm1 = x
bitmap_granularity_bm2 = y

bitmaps can have the same names for different images
bitmap_granularity_bm1_data1 = x
bitmap_granularity_bm1_data2 = y